decidim-admin/app/helpers/decidim/admin/exports_helper.rb

```diff
@@ -10,8 +10,20 @@ module ExportsHelper
       #           current component.
       #
       # Returns a rendered dropdown.
-      def export_dropdown(component = current_component, resource_id = nil)
-        render partial: "decidim/admin/exports/dropdown", locals: { component:, resource_id: }
+      def export_dropdown(component = current_component, resource_id = nil, apply_search: false)
+        filters =
+          if apply_search && respond_to?(:query)
+            { id_in: query.result.map(&:id) }
+          else
+            {}
+          end
+        render_dropdown(component:, resource_id:, filters:)
+      end
+
+      def export_dropdowns(query, component = current_component, resource_id = nil)
+        return export_dropdown(component, resource_id, apply_search: false) if query.conditions.empty?
+
+        export_dropdown(component, resource_id, apply_search: false).concat(export_dropdown(component, resource_id, apply_search: true))
```

Comment:
I cannot say i am a big fan of this kind of behavior. I would have a single export button which may say "Export selection"

Reply:
This change can be done, but before doing that, I would like to note that in decidim-initiatives, both of selection and all items are being shown when filtration exist. I  tried to comply with the conventions applied there, but if you still suggest, it can be done